common/src/main/scala/com/gu/recipeasy/models/Statistics.scala

```diff
@@ -20,3 +20,43 @@ case object PStatsBiggestDayCount extends PersonalStatisticsPoint
 case object PStatsRanking extends PersonalStatisticsPoint
 
 case class DayActivityDistribution(date: String, curationCount: Int, verificationCount: Int, confirmationCount: Int)
+
+case class LeaderboardEntry(userEmail: String, curationCount: Int, verificationCount: Int, confirmationCount: Int) {
+  def total(): Int = {
+    curationCount + verificationCount + confirmationCount
+  }
+}
+
+class Leaderboard(data: List[LeaderboardEntry]) {
```

Comment:
True. Thanks for suggesting that. This went through few iterations and I was organising my mind around the `Leaderboard` class but you are right, it's not actually needed.